### ISSUE TITLE
Selective sync continue to sync parents directory of selected directories

### ISSUE THREAD

**Issue by jsalatiel** [owncloud/client#3361]
Expected behaviour

If i have the following structure:

BUGS/
├── FILE_IN_ROOT
├── FOLDER1
│   └── FILE_IN_FOLDER_1
└── FOLDER2
    └── FILE_IN_FOLDER_2
and in "Choose what to sync" option i check only FOLDER2, FILE_IN_ROOT should not be synced. Only the FILE_IN_FOLDER_2 should be.

Actual behaviour
All files in parent root directory are being synced.

Steps to reproduce
Create the simple tree above,  uncheck BUGS in what to sync , wait to sync , check only FOLDER2, and wait to sync.

Server configuration

Operating system: Ubuntu 14.04 LTS

Web server: apache 2.4

Database: mysql

PHP version: 5.5.9-1ubuntu4.9

ownCloud version: 8.0.2-8

Storage backend: disk
Client configuration

Client version: 1.8.2

Operating system: Linux Mint 17.1 rebecca

OS language: English

Installation path of client: /usr/bin/owncloud

**Comment by MTRichards** [comment]
Interesting point. Though I have to admit this was by design when the selective sync feature was implemented (it follows the Dropbox paradigm). 

Unlike Dropbox though, it is possible to achieve what you are talking about by setting up a sync link only to folder 2 from the desktop sync client. While a little more time consuming to set up, you can set up sync with specific folders in a tree, as long as you are not trying to sync both a parent and a child with separate folder entries in the desktop client.
Does that help any?

**Comment by jsalatiel** [comment]
Are you telling me that , in dropbox, if i use 'selective sync' it will sync the files from the root of parent directory ? Its been a while since i tried dropbox, but i will make a test. Though IMHO selective sync should sync only the folders and subfolders i have selected, nothing else. ( and i am a technical user).
I can use the "add folder" and sync in another location, but i still think this is not the better approach for the end user.

**Comment by MTRichards** [comment]
Pretty much, yes. Though, I have to admit that your comment had me thinking, just to be sure, so I tested. Works that way at this time on my setup. Content that are not folders in parent files are synced down when any child directory is selected.

**Comment by jsalatiel** [comment]
Hi @MTRichards ... interesting ... i know sometimes it is nice to use some well-known paradigm, but i  am still trying to understand if dropbox is doing a good job with this behaviour.  All people i know that uses selective sync in owncloud think this is a bug, actually when i opened this issue i really thought this was a bug too.

Anyway, thank you for your explanation!

**Comment by Lbryson-lifesize** [comment]
agree: if dropbox behaves this way, it is poor behavior.  Owncloud should include only those objects that I select for sync, and not sister objects.

**Comment by BloodyIron** [comment]
Yeah I'm still having this issue. Syncing files at the root when intentionally deselecting the root, is scenario that is already leading to accidental deletions of files. Really need to see a fix here. My clients aren't exactly happy.

**Comment by michaelstingl** [comment]
Found today the settings in OneDrive:

![microsoft_onedrive](https://user-images.githubusercontent.com/214010/48512446-c9318800-e859-11e8-9068-8c18dcd667ef.png)

They have a separate checkbox for all files that are in the parent folder. (siblings)

But I'd like to get rid of the selective sync UI anyway. With VFS it's more intuitive to configure directly in the tree of the file manager.

**Comment by guruz** [comment]
> They have a separate checkbox for all files that are in the parent folder. (siblings)

[FYI that's similar to what I proposed here for other reasons](https://github.com/owncloud/client/issues/3840).

**Comment by MarcoBisi0** [comment]
Old post, but the client still works as described above.
Dirty hack: created a new folder, moved to it the files in parent, deselected the folder in selective sync

2.10.1 owncloud desktop client, ubuntu 20.04
